README.md

```diff
@@ -1,2 +1,185 @@
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Html structure**
+- **Styling css with color**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Google Chrome (or any browser)
+ - Git
+ - Visual studio code (or other code editors)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+> Opend GitBash in your computer and type this (replace 'my-folder' with the name of your desired folder)
+```sh
+  cd my-folder
+```
+```sh
+  git clone git@github.com:Pi1998/hello-microverse.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+```sh
+  code .
+```
+
+ 
+### Install
+
+> You don't need to install anything in your computer since it is a webpage. Instead, you can clone this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Shinn Thant Swam Ye**
+
+- GitHub: [@Pi1998](https://github.com/Pi1998)
+- LinkedIn: [Shinn Thant Swam Ye](https://www.linkedin.com/in/shinn-thant-swam-ye-617a66253/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **[CSS upgrades]**
+- [ ] **[JavaScript Functions]**
+- [ ] **[Accessibility Features]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+https://github.com/Pi1998/hello-microverse/issues
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, please support me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank to the Microverse community along with friends and family for the continuous support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+
+- **[Can I use your project as reference?]**
+
+  - [You are warmly welcome to use our project.]
+
+- **[Is this the final version of your project?]**
+
+  - [No, it is not a final version. We will update and upgrade this project for better usability and visualization.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Excellent job with README 🥇🥇 Please make sure that the LICENSE link is not broken. Right now, I'm getting the following error 👇You can go through [this article](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how you can add a LICENSE to a repository. Kindly fix this ⚒

![image](https://user-images.githubusercontent.com/106119105/234323214-0aaac3a1-6715-40ad-9667-d408e672c4dc.png)